src/sentry/toolbar/iframe_view.py

```diff
@@ -0,0 +1,12 @@
+from django.utils.decorators import method_decorator
+from django.views.decorators.csrf import csrf_exempt
+
+from sentry.web.frontend.base import BaseView
+
+
+class IframeView(BaseView):
+    @method_decorator(csrf_exempt)
+    def get(self, request, organization_slug):
```

Comment:
Do we need `csrf_exempt` for a `get` handler?

Reply:
GET requests don't check CSRF tokens.